File: apps/chat-engine/chat_engine_response.py
class ChatEngineResponse:

  def __init__(self, source_message_id, conversation_id, message):
    self.source_message_id = source_message_id
    self.conversation_id = conversation_id
    self.message = message

  def get_source_message_id(self):
    return self.source_message_id
  
  def get_conversation_id(self):
    return self.conversation_id
  
  def get_message(self):
    return self.message

File: apps/chat-web/src/main/java/com/chatweb/webapp/ai/ChatEngineResponse.java
package com.chatweb.webapp.ai;

import com.fasterxml.jackson.annotation.JsonProperty;

public class ChatEngineResponse {

  @JsonProperty("source_message_id")
  private Long sourceMessageId;

  @JsonProperty("conversation_id")
  private Long conversationId;

  @JsonProperty("message")
  private String message;

  public Long getSourceMessageId() {
    return sourceMessageId;
  }

  public void setSourceMessageId(Long sourceMessageId) {
    this.sourceMessageId = sourceMessageId;
  }

  public Long getConversationId() {
    return conversationId;
  }

  public void setConversationId(Long conversationId) {
    this.conversationId = conversationId;
  }

  public String getMessage() {
    return message;
  }

  public void setMessage(String message) {
    this.message = message;
  }
}
